### ISSUE TITLE
Demo not working

### ISSUE THREAD

**Issue by KaustubhD** [accordproject/markdown-editor#90]
**Describe the bug**
After the `yarn start` and navigating to `localhost:3001/examples`, a blank screen is presented.

**To Reproduce**
1. Go to 'localhost:3001'

**Expected behavior**
The posted screenshot in README.md

**Screenshots**
![image](https://user-images.githubusercontent.com/22456687/65376035-03a95400-dcb9-11e9-96be-4a2988a97073.png)

**Desktop (please complete the following information):**
 - OS: [MacOS]
 - Browser [chrome]
 - Version [76]

**Comment by dselman** [comment]
@irmerk this seems to be an issue with the Formatting Toolbar, can you take a look please?

```
[Error] TypeError: undefined is not an object (evaluating 'styled_components__WEBPACK_IMPORTED_MODULE_3___default.a.div')
	Eval Code (index.js:99)
	eval
	./src/FormattingToolbar/index.js (bundle.js:9219)
	__webpack_require__ (bundle.js:20)
	Eval Code
	eval
	./src/SlateAsInputEditor/index.js (bundle.js:9302)
	__webpack_require__ (bundle.js:20)
	Eval Code
	eval
	./examples/src/index.js (bundle.js:108)
	__webpack_require__ (bundle.js:20)
	Eval Code
	eval
	(anonymous function) (bundle.js:9552)
	__webpack_require__ (bundle.js:20)
	(anonymous function) (bundle.js:84)
	Global Code (bundle.js:85)
```

**Comment by irmerk** [comment]
It looks like this has to do with having `styled-components` in both the `peerDependencies` and `devDependencies`. I think we haven't looked into the `markdown-editor` demo since we made the dependencies this way and were developing in `template-studio-v2`.

I won't have bandwidth to look into this until late this week at least.

**Comment by irmerk** [comment]
Sorry for the delay @KaustubhD, there should be a fix for this now [here as version v0.5.16](https://github.com/accordproject/markdown-editor/releases/tag/v0.5.16)

**Comment by irmerk** [comment]
This issue is persisting in `cicero-ui` now. The fix I made in #91 worked when used in `template-studio-v2`, but is not working in `cicero-ui`. Reopening and adding more labels.

**Comment by irmerk** [comment]
Should be finished now with [this PR](https://github.com/accordproject/cicero-ui/pull/147) being merged.
